packages/bpk-component-calendar/src/BpkCalendarNav.js

```diff
@@ -37,31 +37,33 @@ import STYLES from './BpkCalendarNav.module.scss';
 
 const getClassName = cssModules(STYLES);
 
-const changeMonth = ({ month, min, max, callback, source }) => event => {
-  // Safeguard for disabled buttons is due to React bug in Chrome: https://github.com/facebook/react/issues/8308
-  // PR: https://github.com/facebook/react/pull/8329 - unresolved as of 22/12/2016
-  if (isWithinRange(month, { start: min, end: max })) {
-    event.persist();
-    callback(event, { month, source });
-  }
-};
+const changeMonth =
+  ({ callback, max, min, month, source }) =>
+  (event) => {
```

Comment:
Not to be addressed in this PR: Do we need to review this issue? GH suggests the issue has been fixed

Reply:
Yeah possibly, I notice that it might just be a browser thing or still an issue here: https://github.com/facebook/react/issues/7711